### PR TITLE
Add missing tutorials to documentation classes

### PR DESCRIPTION
**Pull request by Calinou**

### CODE REVIEW

**Review by akien-mga — APPROVED** [review]

File: platform/web/doc_classes/EditorExportPlatformWeb.xml [hunk 1]
```diff
@@ -7,6 +7,7 @@
 	</description>
 	<tutorials>
 		<link title="Exporting for the Web">$DOCS_URL/tutorials/export/exporting_for_web.html</link>
+		<link title="Web documentation index">$DOCS_URL/latest/tutorials/platform/web/index.html</link>
```
Comment (akien-mga) [inline platform/web/doc_classes/EditorExportPlatformWeb.xml:10]:
```suggestion
		<link title="Web documentation index">$DOCS_URL/tutorials/platform/web/index.html</link>
```

Comment (akien-mga) [reply platform/web/doc_classes/EditorExportPlatformWeb.xml:10]: Ping @Calinou

**Comment by akien-mga** [comment]
Thanks!

**Comment by YuriSizov** [comment]
Cherry-picked for 4.1.3. I didn't check if every page exists, but this seems correct at a glance. Importers aren't documented in 4.1, though, so those changes are omitted.

**Pull request merged**